### PR TITLE
135 meta3 jen7

### PR DESCRIPTION
**Pull request by jenlampton**
Another PR for https://github.com/backdrop/backdrop-issues/issues/135

### CODE REVIEW

**Comment by quicksketch** [comment]
Looks like this TravisCI failure was due to an issue with the test machine. I've restarted this build.

File: core/modules/locale/locale.module [hunk 1]
```diff
@@ -674,7 +674,9 @@ function locale_get_plural($count, $langcode = NULL) {
   if (!isset($plural_indexes[$langcode][$count])) {
     // Retrieve and statically cache the plural formulas for all languages.
     if (empty($plural_formulas)) {
-      $plural_formulas = variable_get('locale_translation_plurals', array());
+      foreach (language_list() as $installed_language) {
```
Comment (quicksketch) [inline core/modules/locale/locale.module]: This is the only actual code change in this PR, and it seems to be causing a test failure.

**Comment by jenlampton** [comment]
I'm guessing it's the ->language from the d7 patch, needs to be ->langcode.  Updating locally and running some tests to see.

**Comment by jenlampton** [comment]
Language tests now passing...  going to push the branch to see if all the others do too.

**Comment by jenlampton** [comment]
Removed the language change from this patch - these should be just the docs changes now.

**Pull request merged**